apps/builder/app/builder/features/style-panel/sections/advanced/advanced.tsx

```diff
@@ -19,6 +19,7 @@ import {
 import { Add } from "./add";
 import { CollapsibleSection } from "../../shared/collapsible-section";
 import { visualProperties } from "../sections";
+import { toKebabCase } from "../../shared/keyword-utils";
```

Comment:
why we don't use change-case for it?

Reply:
why do we have our own utility? I thought its because its css specific

Reply:
then the name is too broad

Reply:
so should we rename the one in the css utils?

Reply:
didn't you add those utils?

Reply:
have no idea, next time though